uint8 MODE_VELOCITY=0
uint8 MODE_ABSOLUTE=1

uint8 mode
float64 target_angle   # absolute angle in radians
float64 angular_vel        # angular vel rad/s